plugins/frontend-search/README.md

```diff
@@ -13,54 +12,60 @@ plugins=( ... frontend-search)
-$ angularjs dependency injection
+$ frontend angular dependency injection
+# Will turn into ...
+$ angular dependency injection
 ```
 
 Available search contexts are:
 
-| context       | URL                                                                      |
-|---------------|--------------------------------------------------------------------------|
-| angular(>=2.0)| `https://angular.io/?search=`                                             |
-| angularjs(1.x)| `https://google.com/search?as_sitesearch=angularjs.org&as_q=`
-| aurajs        | `http://aurajs.com/api/#stq=`                                            |
-| bem           | `https://google.com/search?as_sitesearch=bem.info&as_q=`                 |
-| bootsnipp     | `https://bootsnipp.com/search?q=`                                         |
-| caniuse       | `https://caniuse.com/#search=`                                            |
-| codepen       | `https://codepen.io/search?q=`                                            |
-| compassdoc    | `http://compass-style.org/search?q=`                                     |
-| cssflow       | `http://www.cssflow.com/search?q=`                                       |
-| dartlang      | `https://api.dartlang.org/apidocs/channels/stable/dartdoc-viewer/dart:`  |
-| emberjs       | `https://emberjs.com/api/#stp=1&stq=`                                     |
-| fontello      | `http://fontello.com/#search=`                                           |
-| html5please   | `http://html5please.com/#`                                               |
-| jquery        | `https://api.jquery.com/?s=`                                             |
-| lodash        | `https://devdocs.io/lodash/index#`                                       |
-| mdn           | `https://developer.mozilla.org/search?q=`                                |
-| npmjs         | `https://www.npmjs.com/search?q=`                                        |
-| qunit         | `https://api.qunitjs.com/?s=`                                            |
-| reactjs       | `https://google.com/search?as_sitesearch=facebook.github.io/react&as_q=` |
-| smacss        | `https://google.com/search?as_sitesearch=smacss.com&as_q=`               |
-| stackoverflow | `https://stackoverflow.com/search?q=`                                     |
-| unheap        | `http://www.unheap.com/?s=`                                              |
+| context       | URL                                                                         |
+| ------------- | --------------------------------------------------------------------------- |
+| angular       | `https://angular.io/docs/ts/latest/api/#!?query=`                           |
+| angularjs     | `https://google.com/search?as_sitesearch=angularjs.org&as_q=`               |
+| bem           | `https://google.com/search?as_sitesearch=bem.info&as_q=`                    |
+| bootsnipp     | `http://bootsnipp.com/search?q=`                                            |
```

Comment:
Why are some of these changed to `http:`?

Reply:
Some of the URL's are not using `https` anymore, so I had to change few of them to keep them working as expected.

Reply:
I've tested: `bootsnipp`, `caniuse`, `codepen`, `emberjs`, and `stackoverflow`, and they all work with https. 

Except Ember, which doesn't work with neither http nor https due to a change in the API docs (so the whole URL should be changed). I've tested `https://www.google.com/search?as_sitesearch=api.emberjs.com&as_q=` and it seems to do the trick. Remember to update the URL in the plugin.zsh file as well.

Reply:
Solved on the latest changes